server/src/main/java/org/elasticsearch/search/DocValueFormat.java

```diff
@@ -454,7 +454,7 @@ public double parseDouble(String value, boolean roundUp, LongSupplier now) {
             try {
                 n = format.parse(value);
             } catch (ParseException e) {
-                throw new RuntimeException(e);
+                throw new RuntimeException("Cannot parse the value [" + value + "] using the pattern [" + pattern + "]");
```

Comment:
Maybe add `, e` just so we don't lose it?

Reply:
Oh, good catch!